lmdeploy/model.py

```diff
@@ -716,6 +716,79 @@ def messages2prompt(self, messages, sequence_start=True):
         return ret
 
 
+@MODELS.register_module(name='yi')
+class Yi(BaseModel):
+    """Chat template of Yi model."""
+
+    def __init__(self,
+                 system='<|im_start|>system\n',
+                 meta_instruction=None,
+                 user='<|im_start|>user\n',
+                 eoh='<|im_end|>\n',
+                 eoa='<|im_end|>\n',
+                 eosys='<|im_end|>\n',
+                 assistant='<|im_start|>assistant\n',
+                 stop_words=['<|im_end|>', '<|endoftext|>'],
```

Comment:
Can we handle two `stop_words`

Reply:
Yes, we can handle that.

Reply:
Nothing special needs to do when converting the model?

Reply:
The latest Yi models are in llama format.

Reply:
Nothing special. Even if it is the 200k model.